plugins/builders/sidebar.rb

```diff
@@ -1,23 +1,29 @@
 class Builders::Sidebar < SiteBuilder
   def build
     generator do
-      site.data.help_sidebar = build_resources(@site.data.sidebar.help)
+      site.data.help_sidebar = build_resources(@site.data.sidebars.help.resources, @site.data.sidebars.help.collection_name, @site.data.sidebars.help.root)
+      site.data.guides_sidebar = {}
+      site.data.sidebars.guides.each do |specification_name, list|
+        list.each do |version_name, attributes|
+          site.data.guides_sidebar["#{specification_name}_#{version_name}"] = build_resources(attributes.resources, attributes.collection_name, attributes.root)
+        end
+      end
     end
   end
 
-  def build_resources(list)
+  def build_resources(list, collection, root)
```

Comment:
`list` and `collection` are very generic names, difficult to understand and distinguish. Do you have more explicit names in mind?

Reply:
`Collection` seems to be a Bridgetown word. We can just explicit `list`